src/types/templates.ts

```diff
@@ -0,0 +1,5 @@
+import { VenueTemplate } from "./venues";
+
+export const TemplatesWithoutBackButton: VenueTemplate[] = [
+  VenueTemplate.jazzbar,
+];
```

Comment:
This probably belongs in `src/settings` along with all of the other similar 'arrays of venue types' configurations, rather than here in `types/*`

Reply:
There is a bunch of comments in `settings.ts` that they should be refactored:

```// @debt Refactor this constant into types/templates or similar?```